SCons/Node/NodeTests.py

```diff
@@ -592,7 +592,7 @@ def my_contents(obj):
             node = SCons.Node.Node()
             node._func_get_contents = 4
             result = node.get_csig()
-            assert result == '550a141f12de6341fba65b0ad0433500', result
+            assert result == '550a141f12de6341fba65b0ad0433500' or result == '9a3e61b6bcc8abec08f195526c3132d5a4a98cc0' or result == '3538a1ef2e113da64249eea7bd068b585ec7ce5df73b2d1e319d8c9bf47eb314', result
```

Comment:
might be cleaner as `result in ( hash1, hash2, hash3) ` ? (where hash# is the quoted hashes above)

Reply:
I will change it to the result-in syntax.